rustls/src/msgs/enums.rs

```diff
@@ -1,7 +1,10 @@
 #![allow(clippy::upper_case_acronyms)]
 #![allow(non_camel_case_types)]
 /// This file is autogenerated.  See https://github.com/ctz/tls-hacking/
-use crate::msgs::codec::{Codec, Reader};
+use crate::{
```

Comment:
Style nit: we use a one line per module import style, so this should be two separate `use crate::..` lines.